tests/raw-loader.test.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable */
+import fs from 'fs';
+import { runLoaders } from 'loader-runner';
+import Promise from 'bluebird';
+import { getFixtureResource, getFixture, getLoader } from './test-utils';
+
+const runLoadersPromise = Promise.promisify(runLoaders);
+const readFilePromise = Promise.promisify(fs.readFile, { context: fs });
+
+const loaders = getLoader();
+
+describe('Raw Loader: Fixture: basic', () => {
+  const fixtureName = 'basic.scss';
+
+  beforeEach(() => {
+
+  });
+
+  test('should load scss file', async () => {
+    const { result } = await runLoadersPromise({ resource: getFixtureResource(fixtureName), loaders });
```

Comment:
Maybe just two simple tests (default/escaping(#36)) with `loader.call(ctx, content)` for now as the `raw-loader` doesn't demand any requirements and proper testing in a follow up

Reply:
@michael-ciniawsky - Yeah, I was just tinkering with Seans test setup which is complete overkill here but how cumbersome it was in a small repo was what I was curious about.

I'm going to lean this out a bunch more inline with your initial pass